projects/core/tokens/svg-src-processor.ts

```diff
@@ -0,0 +1,7 @@
+import {InjectionToken} from '@angular/core';
+import {identity, TuiStringHandler} from '@taiga-ui/cdk';
+
+export const TUI_SVG_SRC_PROCESSOR: InjectionToken<TuiStringHandler<string>> =
+    new InjectionToken<TuiStringHandler<string>>('Source path processor for svg', {
+        factory: () => identity,
```

Comment:
Let's use `identity` from rxjs and deprecate our `identity`.

Reply:
fixed